esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -3097,16 +3045,13 @@ bool CFileSprayEx::onOpenSave(IEspContext &context, IEspOpenSaveRequest &req, IE
     return true;
 }
 
-bool CFileSprayEx::getDropZoneFiles(IEspContext &context, const char* netaddr, const char* osStr, const char* path,
+bool CFileSprayEx::getDropZoneFiles(IEspContext &context, const char* dropZone, const char* netaddr, const char* path,
```

Comment:
The dropzone was added to support the server list inside dropzone. In old environment, one dropzone has one IP. So, the assumption was that the IP can be used to identify a dropzone, although that assumption may not be correct. Now, for new environment, dropzone has to be used to define a dropzone.